databricks/koalas/groupby.py

```diff
@@ -2343,6 +2344,73 @@ def get_group(self, name) -> Union[DataFrame, Series]:
 
         return DataFrame(internal)
 
+    def median(self, numeric_only=True, accuracy=10000) -> Union[DataFrame, Series]:
```

Comment:
Are `numeric_only` and `accuracy` unknown types?

Reply:
We should add more type hints for function arguments in the separate PR.

Reply:
Sure, I'll add in the separated PR.

Thanks for the review!

Reply:
Oh, anyway `numeric_only` only supports for `True` since this is only for pandas compatibility.

Should we add type hints as `bool` though ?

Reply:
Yep, it must be `bool`, and `accuracy` must be `int`?
There are other similar places in the file or the other files, we can do it in a batch.

Reply:
Sounds good. I will take a look and add type hints throughout the whole file next week !